src/components/Dialog.js

```diff
@@ -1,8 +1,14 @@
 import classnames from 'classnames';
 import { useEffect, useLayoutEffect, useRef, useState } from 'preact/hooks';
 
-import { IconButton, LabeledButton } from '../';
-import { SvgIcon } from './SvgIcon';
+import { IconButton, LabeledButton } from './buttons';
+import { registerIcons, SvgIcon } from './SvgIcon';
+
+// Register the checkbox icon for use
+registerIcons({
+  /** @ts-ignore - TS doesn't understand require here */
+  'hyp-cancel': require('../../images/icons/cancel.svg'),
+});
```

Comment:
When trying out the `Dialog` in my local `lms`, it broke because it assumed the presence of a `cancel` icon. Whoops! We'd already run into this before with the `Checkbox` component, and I overlooked making sure other components registered their own icons. We're using the `hyp-` prefix here to avoid icon collisions.